### PR TITLE
Fixed typos in javadoc

### PR DESCRIPTION
**Pull request by tivervac**

### CODE REVIEW

File: org.eclipse.lsp4j/src/main/java/org/eclipse/lsp4j/Protocol.xtend [hunk 1]
```diff
@@ -4938,13 +4938,13 @@ class ConfigurationParams {
 
 /**
  * A ConfigurationItem consist of the configuration section to ask for and an additional scope URI.
- * The configuration section ask for is defined by the server and doesn’t necessarily need to
- * correspond to the configuration store used be the client. So a server might ask for a configuration
- * cpp.formatterOptions but the client stores the configuration in a XML store layout differently.
+ * The configuration section asked for is defined by the server and doesn’t necessarily need to
+ * correspond to the configuration store used by the client. So a server might ask for a configuration
+ * {@code cpp.formatterOptions} but the client stores the configuration in an XML store layout differently.
```
Comment (cdietrich) [inline org.eclipse.lsp4j/src/main/java/org/eclipse/lsp4j/Protocol.xtend:4943]: an is wrong (no vocal)

File: org.eclipse.lsp4j/src/main/xtend-gen/org/eclipse/lsp4j/ConfigurationItem.java [hunk 2]
```diff
@@ -16,13 +16,13 @@
 
 /**
  * A ConfigurationItem consist of the configuration section to ask for and an additional scope URI.
- * The configuration section ask for is defined by the server and doesn’t necessarily need to
- * correspond to the configuration store used be the client. So a server might ask for a configuration
- * cpp.formatterOptions but the client stores the configuration in a XML store layout differently.
+ * The configuration section asked for is defined by the server and doesn’t necessarily need to
+ * correspond to the configuration store used by the client. So a server might ask for a configuration
+ * {@code cpp.formatterOptions} but the client stores the configuration in an XML store layout differently.
```
Comment (cdietrich) [inline org.eclipse.lsp4j/src/main/xtend-gen/org/eclipse/lsp4j/ConfigurationItem.java:21]: should be a, no vocal

**Comment by KamasamaK** [comment]
The javadoc is mostly verbatim from the actual spec. Consider making a PR in [that repo](https://github.com/microsoft/language-server-protocol) too.

**Comment by tivervac** [comment]
See https://github.com/microsoft/language-server-protocol/pull/1081

**Review by KamasamaK — APPROVED** [review]
The changes match the spec now except for the "be" to "by" change, but that is a proper correction. Please make that change in the spec as well. LGTM

**Comment by tivervac** [comment]
@KamasamaK See https://github.com/microsoft/language-server-protocol/pull/1090

**Pull request merged**